tools/pot/tests/test_graph_with_stats.py

```diff
@@ -15,24 +15,26 @@
 from .utils.config import merge_configs
 
 TEST_MODELS = [
-    ('resnet_example', 'pytorch')
+    ('resnet_example', 'pytorch', 'Ranger', 'ANY'),
+    ('act_act_example', 'pytorch', 'DefaultQuantization', 'GNA')
```

Comment:
What does `act_act_example` mean? Is about `input_output_example`

Reply:
Sorry for the confusing name. I will change it. 
Originally, `act_act` means `activation & activation` pattern. To be truth, this model contains both needed patterns: parameter + activation and activation + activation